pkg/artifactory/resource/security/resource_artifactory_scoped_token.go

```diff
@@ -0,0 +1,285 @@
+					"* applied-permissions/groups - the group to which permissions are assigned by group name (use username to inicate the group name)" +
+					"* system:metrics:r - for getting the service metrics" +
+					"* system:livelogs:r - for getting the service livelogsr" +
+					"The scope to assign to the token should be provided as a list of scope tokens, limited to 500 characters in total.\n" +
+					"Resource Permissions\n" +
+					"From Artifactory 7.38.x, resource permissions scoped tokens are also supported in the REST API. A permission can be represented as a scope token string in the following format:\n" +
+					"`<resource-type>:<target>[/<sub-resource>]:<actions>`\n" +
+					"Where:\n" +
+					"* `<resource-type>` - one of the permission resource types, from a predefined closed list. Currently, the only resource type that is supported is the artifact resource type.\n" +
+					"* `<target>` - the target resource, can be exact name or a pattern" +
+					"* `<sub-resource>` - optional, the target sub-resource, can be exact name or a pattern" +
+					"* `<actions>` - comma-separated list of action acronyms." +
+					"The actions allowed are <r, w, d, a, m> or any combination of these actions\n." +
+					"To allow all actions - use `*`",
+			},
+			"expires_in": {
+				Type:             schema.TypeInt,
+				Optional:         true,
+				ForceNew:         true,
+				Computed:         true,
+				ValidateDiagFunc: validator.IntAtLeast(0),
+				Description:      "The amount of time, in seconds, it would take for the token to expire. An admin shall be able to set whether expiry is mandatory, what is the default expiry, and what is the maximum expiry allowed. Must be non-negative. Default value is based on configuration in 'access.config.yaml'. See [API documentation](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-RevokeTokenbyIDrevoketokenbyid) for details.",
+			},
+			"refreshable": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				ForceNew:    true,
+				Description: "The token is not refreshable by default.",
+			},
+			"description": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				ForceNew:         true,
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringLenBetween(0, 1024)),
+				Description:      "Free text token description. Useful for filtering and managing tokens. Limited to 1024 characters.",
+			},
+			"audiences": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				ForceNew: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Description: "A list of the other instances or services that should accept this token identified by their Service-IDs. Limited to total 255 characters. Default to '*@*' if not set. For instructions to retrieve the Artifactory Service ID see this [documentation](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-GetServiceID).",
```

Comment:
You say it defaults, but I don't see that set here

Reply:
There is a pattern here that can be enforced

Reply:
Because TF SDK doesn't support `Default` or `DefaultFunc` with non-primitive type.